packages/devtools/src/schema/typeDefinitions.ts

```diff
@@ -49,4 +49,13 @@ directive @createModel(
 ) on OBJECT
 
 directive @loadModel(id: StreamID!) on OBJECT
+
+input IndexField {
+  path: [String!]!
+}
+
+directive @createIndex(
+  name: String!
```

Comment:
Can we make the `name` optional please?
Also, what error message gets raised on duplicate names? Is is properly caught by the tools?